ansible/roles/common/tasks/main.yml

```diff
@@ -50,6 +51,17 @@
     name: supervisor
     enabled: yes
 
+- name: Permit password sshd auth
+  lineinfile:
+    path: /etc/ssh/sshd_config
+    regexp: "^[#]?PasswordAuthentication "
+    line: "PasswordAuthentication yes"
+
+- name: Disable ssh server
```

Comment:
But we want the sshd enabled at all times, because even in production, the users can set up their own ssh keys through the admin UI.

Reply:
:+1: